crrExistingObjects.js

```diff
@@ -302,6 +304,7 @@ function triggerCRROnBucket(bucketName, cb) {
 series([
     next => metadataUtil.metadataClient.setup(next),
     next => eachSeries(BUCKETS, triggerCRROnBucket, next),
+    next => metadataUtil.metadataClient.close(next),
```

Comment:
Curious, did `metadataClient.close()` work? When I was working on this, it threw an error IIRC! Also, checked other repos where metadataClient was used and `metadataClient.close()` was not used, I might be wrong!

Reply:
works for me :)